src/data/matrix/basic.lean

```diff
@@ -303,58 +303,60 @@ variable [fintype m]
 def dot_product [has_mul α] [add_comm_monoid α] (v w : m → α) : α :=
 ∑ i, v i * w i
 
+localized "infix  ` ⬝ᵥ `:67 := matrix.dot_product" in matrix
```

Comment:
Can you add a comment justifying the choice of 67?

Reply:
I changed it to 72 and explained.